app/mailers/conversation_reply_mailer.rb

```diff
@@ -79,8 +79,8 @@ def last_outgoing_message
     @conversation.messages.chat.where.not(message_type: :incoming)&.last
   end
 
-  def assignee_name
-    @assignee_name ||= @agent&.available_name || 'Notifications'
+  def sender_name
+    @sender_name ||= @message&.sender&.name || @agent&.available_name || 'Notifications'
```

Comment:
```   suggestion
@sender_name ||= @message&.sender&.name || @agent&.available_name || I18n.t('conversations.reply.email.sender.default')
```

@tejaswinichile  Should `Notifications` be read from i18n to better support localization?

Reply:
@vishnu-narayanan `Notifications` case never happens until someone changes the DB validation on not to save the name. I have kept it because it was always there from the start.

And we have the next PR to handle the whole sender name customization.